mm2src/coins/z_coin.rs

```diff
@@ -284,6 +284,10 @@ impl ZCoin {
     #[inline]
     pub fn consensus_params_ref(&self) -> &ZcoinConsensusParams { &self.z_fields.consensus_params }
 
+    pub async fn is_sapling_state_synced(&self) -> bool {
```

Comment:
Can you add this change with different commit? like "add `fn is_sapling_state_synced` for ZCoin"